AppAuth-JS/built/errors.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork